src/main/scala/com/microsoft/partnercatalyst/fortis/spark/ProjectFortis.scala

```diff
@@ -128,6 +129,16 @@ object ProjectFortis extends App {
           )
         )
       ),
+      "bing" -> List(
+        ConnectorConfig(
+          "BingPage",
+          Map(
+            "accessToken" -> System.getenv("BING_ACCESS_TOKEN"),
+            "searchInstanceId" -> System.getenv("BING_SEARCH_INSTANCE_ID"),
+            "keywords" -> "isis|attack"
```

Comment:
Not a deal-breaker, but may I suggest `Option(System.getenv("BING_SEARCH_KEYWORDS")).getOrElse("isis|attack")` instead of the raw term?